src/components/MultiButtonWithSubQuery.tsx

```diff
@@ -15,7 +15,21 @@ export const navigateWithSubQuery = ({
 }: {
   navigation: StackNavigationProp<any, string>;
   title?: string;
-  params?: { routeName: string; webUrl: string } | string;
+  params?:
+    | {
+        routeName: string;
+        webUrl: string;
+        paramsForMultiButtons?: {
```

Comment:
Why not short `params` here as well?

Reply:
I did not want to use params.params in the code. 

```
 if (!!params && typeof params === 'object') {
    if (params.paramsForMultiButtons) {
        ...
        params: {
          ...params.paramsForMultiButtons,
        }
      });
    }
```
if I use params this code will look like the following.
```
if (!!params && typeof params === 'object') {
    if (params.params) {
      ...
        params: {
          ...params.params,
        }
      });
    }
```

Reply:
Okay, I can rename this prop this way. It'll be shorter and clearer. 👍

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/1fd55b6e7ac650ac6929042ae0f3310262c94971